cvat/apps/analytics_report/report/primary_metrics/annotation_speed.py

```diff
@@ -84,26 +84,24 @@ def get_default():
 
         data_series = statistics["data_series"]
```

Comment:
Fileds of `data_series` are changed in the following code, probably you should use deepcopy.
FYI: it looks like you don't need the `get_default` method, `get_empty` is enough.